metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
@justinleet and I communicated about the bcrypt-pbkdf issue, and I proved to my satisfaction that the 1.0.1 version has a correct and validated change of license from 4-clause to 3-clause BSD.  And I've seen no other license issues raised.  So I'm fine with this.